hypothesis-python/RELEASE.rst

```diff
@@ -1,6 +1,11 @@
 RELEASE_TYPE: patch
 
+<<<<<<< 2a5294f8c4b9879fa6ce9f3a3261c4e4e04cc850
```

Comment:
Merge conflict alert!

Reply:
@DRMacIver - still a merge conflict, because this diff is edits to an existing file.  You need to rebase on a commit where the release file doesn't exist, even though they're not lasting long at HEAD at the moment :wink: